### DISCUSSION TITLE
Are yall still alive?

### DISCUSSION THREAD

**Discussion by Ehren1337** [xoreos/xoreos#602] [General]
Is this still being worked on?

**Comment by PerAstraAdDeum** [comment]
I'd like to pose that question as well. Hope this project isn't dead! :-|

**Comment by jjdredd** [comment]
I'm not aware of any alternatives so the question whether this project is alive or not is imo irrelevant. I've been wanting to contribute, but cant find the time to learn the project and figure out features of the original engine.

**Comment by DrMcCoy** [comment]
Eeeeeh... yes and not, but mostly no. I kinda go into this a bit here: https://github.com/xoreos/xoreos/issues/612#issuecomment-3444448402
